README.md

```diff
@@ -19,17 +19,19 @@ To receive full commercial license from Banuba - please fill in our form on [for
    >`banuba_effect_player-release.aar` => `libs/`
 3. Copy `include/bnb` directory into `libs/bnb_sdk/`:
    >`include/bnb` => `libs/bnb_sdk/bnb`
-4. Get [The latest Agora Video SDK archive for android](https://docs.agora.io/en/extension_customer/Banuba_downloads).
-5. Copy `jar` files from the Agora Video SDK archive into `libs` dir:
+4. Download and copy folders of the needed effects into `app/src/main/assets/effects` directory. 
+The effects can be downloaded from [here](https://docs.banuba.com/face-ar-sdk-v1/overview/demo_face_filters).
+5. Get [The latest Agora Video SDK archive for android](https://docs.agora.io/en/extension_customer/Banuba_downloads).
```

Comment:
please remove the words "The latest" since it can be not the latest but we verified the sample only with that version. Also is there a possibility to avoid this step via downloading that package via Maven?